src/context/transactions-repository.ts

```diff
@@ -47,12 +48,22 @@ export class TransactionRepository extends BaseRepository<TransactionEntity> {
           });
         }
 
+        if (options.where?.transactionId && transactionIds) {
+          transactionIds.push(options.where.transactionId);
+          delete options.where.transactionId;
+        }
+
+        if (transactionIds) {
```

Comment:
If an empty array is submitted, it will result in a broken query, please add a check for it.

Reply:
Fixed.